__tests__/app.test.js

```diff
@@ -37,4 +37,39 @@ describe("/api/app", () => {
         });
       });
   });
+  test("Reviews should have properties of title, designer, review_body, review_id, review_img_url, votes, owner, created_at, category", () => {
+    return request(app)
+      .get("/api/reviews/5")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.review.review_id).toBe(5);
```

Comment:
Perfect addition on line 45 here. Just to say, if this assertion is valid, then we know there must be a key of review_id on the object. That means we can remove line 49 entirely. Also, if we know what review 5 looks like, then we can directly assert against it, rather than just checking its properties.